AfricanReligions/events/afr_misc_events.txt

```diff
@@ -235,4 +235,219 @@ character_event = {
             }
         }
     }
+}
+
+# Bori Selecting Patron Deities events
+character_event = {
+    id = AFRREL.00500
+    desc = EVTDESC_AFRREL00500
+    border = GFX_event_normal_frame_religion
+    picture = GFX_evt_desert
+
+    is_triggered_only = yes
+
+    immediate = {
+        if = {
+            limit = { if_playable = yes }
+            any_courtier = {
+                limit = {
+                    prisoner = no
+                    religion = root
+                    age = 16
+                    has_minor_deity_trigger = no
```

Comment:
If you want this to work, you need to modify this trigger so that it returns true if the character has one of our modifiers.

Reply:
I'm not actually sure what this does I copied the event structure of the Slavic pick patron deity. There was also an identical event just without the immediate bit so idk... could likely just remove it?

Reply:
No, you need it here.  It's a scripted trigger, they are defined in common/scripted_triggers, essentially it is a condition block that can be reused like a function.  But this one only checks for vanilla patron deities, so you'll have to add ours to it, it is used here to fire the patron deity event for all of the player's courtiers that don't have one - the condition block limits the scope to just courtiers that don't have a patron deity.